exercises/say/canonical-data.json

```diff
@@ -1,6 +1,6 @@
 {
   "exercise": "say",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "comments": [
     "Here -1 is used as expected value to indicate that the",
```

Comment:
Would it make sense to update the comments to reflect the new error object?

Reply:
Yes it would.  :smile:   Thanks for pointing it out.